CHANGELOG.md

```diff
@@ -14,11 +14,11 @@
 #### Improvements
 
 - Improved query generation for `Dataframe.distinct` to generate `SELECT DISTINCT` instead of `SELECT` with `GROUP BY` all columns.
+- Improved the random object name generation to avoid collisions.
 
 #### Deprecations
 
 - Deprecated Snowpark Python function `snowflake_cortex_summarize`. Users can install snowflake-ml-python and use the snowflake.cortex.summarize function instead.
-- Deprecated Snowpark Python function `snowflake_cortex_sentiment`. Users can install snowflake-ml-python and use the snowflake.cortex.sentiment function instead.
 
```

Comment:
why is this removed?

Reply:
oh sorry, I thought this was a duplication, adding it back